Dockerfile

```diff
@@ -84,6 +84,13 @@ RUN export GF_GID_NAME=$(getent group $GF_GID | cut -d':' -f1) && \
   chown -R "grafana:$GF_GID_NAME" "$GF_PATHS_DATA" "$GF_PATHS_HOME/.aws" "$GF_PATHS_LOGS" "$GF_PATHS_PLUGINS" "$GF_PATHS_PROVISIONING" && \
   chmod -R 777 "$GF_PATHS_DATA" "$GF_PATHS_HOME/.aws" "$GF_PATHS_LOGS" "$GF_PATHS_PLUGINS" "$GF_PATHS_PROVISIONING"
 
+ENV NI_PLUGINS_VERSION="2022-02-07T234351"
```

Comment:
We'll probably want this to pull in the latest release dynamically right?
This is a good place to start, though.

Reply:
I wonder if we'll even be able to pass that as a parameter if we have one pipeline triggering another pipeline. Either way, I'm fine using the hardcoded version for now.

Reply:
Yeah, I was thinking we could pass the version `--build-args` to the arguments in the docker build step in the pipeline, but I haven't thought through where we would get the value to pass in to it or whether/how that would work with one pipeline triggering another. Something to look in to in the future.

Reply:
That reminds me, I need to make a story capturing the plugin pipeline triggering work. Hardcoding it for now sounds fine (we probably won't make that many changes to the plugins repo anyway).